reps_event_app/lib/ui/reps.dart

```diff
@@ -0,0 +1,260 @@
+        // title: Text("Reps Events Mobile"),
+        elevation: 0,
+      ),
+      drawer: Drawer(
+        child: ListView(
+          padding: EdgeInsets.all(5.0),
+          children: <Widget>[
+            DrawerHeader(
+              child: Container(
+                alignment: Alignment.topLeft,
+                child: Image.asset(
+                  'assets/img/reps.jpg',
+                  width: 80,
+                  height: 80,
+                ),
+              ),
+            ),
+            getList(listTitle: "About"),
+            getList(listTitle: "Events"),
+            getList(listTitle: "People")
+          ],
+        ),
+      ),
+      body: Column(
+        children: <Widget>[getSearchBar(), Expanded(child: getRepsList())],
+      ),
+    );
+  }
+
+  getList({String listTitle}) {
+    return ListTile(
+      title: Text(listTitle),
+      onTap: () {
+        Navigator.pop(context);
+      },
+    );
+  }
+
+  Future<List<RepsModel>> fetchEvent() async {
+    print("FETCH CALLED");
+    List<RepsModel> reps = [];
+    print(dateTime);
+    var response =
+    await fetchReps(date: dateTime, cityName: _searchController.text)
+        .catchError((e) {
+      print(e);
+    });
+    response['objects'].forEach((jsonObject) {
+      reps.add(RepsModel.fromJson(jsonObject));
+    });
+    return reps;
+  }
+
+  getRepsList() {
+    return FutureBuilder<List<RepsModel>>(
+        future: _future,
+        builder: (context, snapshot) {
+          if (snapshot.hasData) {
+            return snapshot.data.isEmpty
+                ? Container(
+              child: Center(
+                child: Image.asset('assets/img/no_results_found.gif'),
+              ),
+            )
+                : ListView.builder(
+                shrinkWrap: true,
+                physics: BouncingScrollPhysics(),
+                scrollDirection: Axis.vertical,
+                itemCount: snapshot.data.length,
+                itemBuilder: (context, index) {
+                  ;
+                  return getEventTile(snapshot, index);
+                });
+          } else {
+            return Container(
+              child: Center(
+                child: Image.asset(
+                  "assets/img/preloader.gif",
+                ),
+              ),
+            );
+          }
+        });
+  }
+
+  Card getEventTile(AsyncSnapshot snapshot, int index) {
+    return Card(
+        child: ListTile(
+          onTap: () {
+            Navigator.push(
+                context,
+                MaterialPageRoute(
+                    builder: (context) => RepsDetails(
+                      reps: snapshot.data[index],
+                    )));
+          },
+          title: Text(
+            snapshot.data[index].fullname,
+            maxLines: 5,
+            overflow: TextOverflow.ellipsis,
+            style: TextStyle(fontFamily: 'Zilla Slab', fontSize: 18.0),
+          ),
+          subtitle:
```

Comment:
No need to display the city in subtitle.
In `ListTile` we need to display Reps profile image & name